README.md

```diff
@@ -217,10 +217,9 @@ For example, if a method expects the unanalyzed term and is called with an analy
 Initialize the class as follows:
 
 ```python
-from pyserini.index import pyutils
-from pyserini.analysis import pyanalysis
+from pyserini import analysis, index
 
-index_utils = pyutils.IndexReaderUtils('indexes/index-robust04-20191213/')
+index_utils = index.IndexReaderUtils('indexes/index-robust04-20191213/')
```

Comment:
For brevity, should this be called `index.Reader()`? or `index.IndexReader()`

Reply:
I like just `IndexReader`. I suppose I'll change it while I'm doing this massive refactoring.